install-server.sh

```diff
@@ -64,6 +65,14 @@ cmdline() {
         read -rp "Please provide the path of the runner private key: " PRIVATE_KEY </dev/tty
     fi
 
+    # Replace tilde with the current home:
+    private_key_absolute_path=$(realpath -q -e "$PRIVATE_KEY")
+    if [[ $? > 0 ]]; then
```

Comment:
I tested it and this does not work since if the file is not present the `if` will never be reached
I would do it this way:

```bash
private_key_absolute_path=$(realpath -q -e "$PRIVATE_KEY") || { echo "Path '${PRIVATE_KEY}' to private SSH key does not exist, please try again."; exit 1; }
```

Reply:
Thanks @fabriziosestito for testing it. That's a bit surprising as I've tried it out in my shel first. Although I have to admit it was just this small snippet and not the whole script.

Anyway, I've added your line. I wasn't sure about line length so I've split it. Let me know if this is okay or if you prefer it in another way.